js/src/carousel.js

```diff
@@ -150,6 +156,16 @@ class Carousel extends BaseComponent {
     }
     // End mod
 
+    // Boosted mod: if a play-pause button is present, set the button to play on mouseenter
+    if (this._playPauseButton !== null && this._playPauseButton.classList.contains('pause')) {
+      this._playPauseButton.classList.remove('pause')
+      this._playPauseButton.classList.add('play')
+      this._playPauseButton.setAttribute('title', this._playPauseButton.getAttribute(SELECTOR_CAROUSEL_PLAY_TEXT))
```

Comment:
Should we get the attributes once in the constructor such as `this._playPauseButton` ?

Reply:
+ I think we should have a default value in English I guess